RocketChat/Rocket.Chat
# [FIX] OEmbedURLWidget - Show Full Embedded Text Description

## Proposed changes (including videos or screenshots)



Embeds were cutoff when either _urls had a long description_.
This was handled by removing `overflow:hidden;text-overflow:ellipsis;` from the inline styles in [`oembedUrlWidget.html`](https://github.com/RocketChat/Rocket.Chat/blob/develop/app/oembed/client/oembedUrlWidget.html#L28).

### Earlier

![earlier](https://user-images.githubusercontent.com/55396651/107110825-00dcde00-6871-11eb-866e-13cabc5b0d05.png)

### Now

![now](https://user-images.githubusercontent.com/55396651/107110794-ca06c800-6870-11eb-9b3b-168679936612.png)



## Issue(s)


Closes #20628

## Steps to test or reproduce


- Send a link to any channel (or direct message)
Make sure the *description of the link is long*. Try `https://95nyn.csb.app/` or `https://github.com`.
- You will find that the text

## Further comments


**This is not related to [Issue #20556](https://github.com/RocketChat/Rocket.Chat/issues/20556#issuecomment-774411671)**
